schema/spacewalk/upgrade/susemanager-schema-4.2.0-to-susemanager-schema-4.2.1/120-arm64.sql

```diff
@@ -0,0 +1,96 @@
+insert into rhnCpuArch (id, label, name) select
+sequence_nextval('rhn_cpu_arch_id_seq'), 'arm64', 'ARM64' from dual
+where not exists (select 1 from rhnCpuArch where label = 'arm64');
+
+
+insert into rhnPackageArch (id, label, name, arch_type_id) select
+sequence_nextval('rhn_package_arch_id_seq'), 'arm64-deb', 'ARM64-deb', lookup_arch_type('deb') from dual
+where not exists (select 1 from rhnPackageArch where label = 'arm64-deb');
+
+
+insert into rhnServerArch (id, label, name, arch_type_id) select
+sequence_nextval('rhn_server_arch_id_seq'), 'arm64-debian-linux', 'ARM64 Debian', lookup_arch_type('deb') from dual
+where not exists (select 1 from rhnServerArch where label = 'arm64-debian-linux');
+
+
+insert into rhnChannelArch (id, label, name, arch_type_id) select
+sequence_nextval('rhn_channel_arch_id_seq'), 'channel-arm64-deb', 'ARM64 Debian', lookup_arch_type('deb') from dual
+where not exists (select 1 from rhnChannelArch where label = 'channel-arm64-deb');
+
+
+insert into rhnPackageUpgradeArchCompat (package_arch_id, package_upgrade_arch_id, created, modified) select 
+LOOKUP_PACKAGE_ARCH('arm64-deb'), LOOKUP_PACKAGE_ARCH('all-deb'), current_timestamp, current_timestamp from dual
+where not exists (select 1 from rhnPackageUpgradeArchCompat where package_arch_id = LOOKUP_PACKAGE_ARCH('arm64-deb') and package_upgrade_arch_id = LOOKUP_PACKAGE_ARCH('all-deb'));
+
+-- insert into rhnPackageUpgradeArchCompat (package_arch_id, package_upgrade_arch_id, created, modified) select 
```

Comment:
Same: why do we need it commented?

Reply:
I would say, these statements are obsolet;
dito the corresponding statements for "amd64"

```
uyuni=# insert into rhnPackageUpgradeArchCompat (package_arch_id, package_upgrade_arch_id, created, modified) select
LOOKUP_PACKAGE_ARCH('arm64-deb'), LOOKUP_PACKAGE_ARCH('arm64-deb'), current_timestamp, current_timestamp from dual
where not exists (select 1 from rhnPackageUpgradeArchCompat where package_arch_id = LOOKUP_PACKAGE_ARCH('arm64-deb') and package_upgrade_arch_id = LOOKUP_PACKAGE_ARCH(arm64-deb'));
uyuni'#
```

Because of the foreign key constraints on table rhnPackageUpgradeArchCompat:
https://github.com/uyuni-project/uyuni/blob/master/schema/spacewalk/common/tables/rhnPackageUpgradeArchCompat.sql
it is not possible to insert the same value twice;